src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern(
```

Comment:
```suggestion
    private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern(
```
якщо клас не незмінний(immutable) або примітив потрібно стандартний неймінг юзати